packages/terra-clinical-result/src/terra-dev-site/doc/example/Examples.module.scss

```diff
@@ -146,8 +145,11 @@
     margin: 0;
     max-height: 100%;
     max-width: 100%;  // Needed for IE10 truncation
-    overflow: hidden; // Needed for IE10
     padding: 0;
     width: 100%;
   }
+
+  .flowsheetresult-minwidth-override--for-testing-only {
```

Comment:
Is this intended to be removed? Why the need to append `--for-testing-only` to the class name?

Reply:
Was preserving a match of the visual tests from previously, when it didn't have a min-Width set, by applying an override (intent was for testing only)

Could be better to remove and do something like this: https://github.com/cerner/terra-clinical/pull/637/files#r399518462